### PR TITLE
[Security] Allow passport attributes in `Security::login()`

### LINKED ISSUE #20229: [Security][SecurityBundle] Allow passing attributes to passport via `Se…
| Q            | A
| ------------ | ---
| Feature PR   | symfony/symfony#58166
| PR author(s) | @alexandre-daubois
| Merged in    | 7.2

We created this issue to not forget to document this new feature. We would really appreciate if you can help us with this task. If you are not sure how to do it, please ask us and we will help you.

To fix this issue, please create a PR against the 7.2 branch in the [symfony-docs repository](https://github.com/symfony/symfony-docs).

Thank you! :smiley:

### PR DESCRIPTION
**Pull request by alexandre-daubois**
Fix #20229

### CODE REVIEW

**Review by javiereguiluz — APPROVED** [review]

File: security.rst [hunk 1]
```diff
@@ -1767,9 +1767,12 @@ You can log in a user programmatically using the ``login()`` method of the
             // you can also log in on a different firewall...
             $security->login($user, 'form_login', 'other_firewall');
 
-            // ...and add badges
+            // ... add badges...
             $security->login($user, 'form_login', 'other_firewall', [(new RememberMeBadge())->enable()]);
 
+            // ... and also add passport attributes
+            $security->login($user, 'form_login', 'other_firewall', [(new RememberMeBadge())->enable()], ['arbitrary_key' => 'some data']);
```
Comment (OskarStark) [inline security.rst]: Let's use some real values here for example. 

Referer maybe?

Comment (alexandre-daubois) [reply security.rst]: I tried to update according to your comment. Is it what you had in mind?

Comment (OskarStark) [reply security.rst]: Perfect, thanks

**Comment by OskarStark** [comment]
Thank you Alexandre.

**Pull request merged**